util/start_test

```diff
@@ -90,7 +90,7 @@ def run_tests(tests):
 
     # print out flags
     logger.write('[tests: "{0}"]'.format(" ".join(files)))
-    if args.recurse:
+    if args.recurse and not os.path.isfile("NORECURSE"):
```

Comment:
I created a `NOTEST` file in `pyzmq-venv/` and this only keeps `start_test` from testing anything within that folder, but it still recurses into the directory tree of the Python venv. It's not a big deal because there aren't `*.chpl` files in that directory tree, but it produces output "cruft" that I'd _prefer_ to avoid if it's not too much trouble.